src/librustdoc/html/render/print_item.rs

```diff
@@ -355,11 +355,10 @@ fn item_module(w: &mut Buffer, cx: &Context<'_>, item: &clean::Item, items: &[cl
                 let doc_value = myitem.doc_value().unwrap_or_default();
                 write!(
                     w,
-                    "<tr class=\"{stab}{add}module-item\">\
-                         <td><a class=\"{class}\" href=\"{href}\" \
-                             title=\"{title}\">{name}</a>{unsafety_flag}</td>\
-                         <td class=\"docblock-short\">{stab_tags}{docs}</td>\
-                     </tr>",
+                    "<item-left class=\"{stab}{add}module-item\">\
+                         <a class=\"{class}\" href=\"{href}\" \
+                             title=\"{title}\">{name}</a>{unsafety_flag}</item-left>\
+                     <item-right class=\"docblock-short\">{stab_tags}{docs}</item-right>\n",
```

Comment:
Why the `\n`?

Reply:
Primarily, to make the output more readable and grepable. But I am open to removing it

Reply:
Please remove it.

Reply:
Done